web/src/components/pages/about/playbook-content/how-funded.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Localized } from '@fluent/react'
+
+const HowFunded = () => (
+  <>
+    <Localized
+      id="about-playbook-how-funded-content-1"
+      elems={{
+        philantropicGrantLink: (
+          <a
+            href="https://foundation.mozilla.org/en/blog/mozilla-common-voice-dataset-grows-by-30-and-reaches-87-languages/"
+            target="_blank"
+            rel="noopener noreferrer"
+          />
+        ),
+      }}>
+      <p />
+    </Localized>
+    <Localized
+      id="about-playbook-how-funded-content-2"
+      elems={{
+        africaMradiLink: (
+          <a
+            href="https://wiki.mozilla.org/Africa_Mradi"
+            target="_blank"
+            rel="noopener noreferrer"
+          />
+        ),
+      }}>
+      <p />
+    </Localized>
+    <Localized
+      id="about-playbook-how-funded-content-3"
+      elems={{
+        donateLink: <a href="?form=FUNUAFTPPYR" />,
+        emailFragment: (
+          <a
+            href="mailto:commonvoice@mozilla.com"
```

Comment:
Probably should use a constant for the mailto email instead of hard coded email here